platform/web/export/export_plugin.h

```diff
@@ -56,11 +56,14 @@ class EditorExportPlatformWeb : public EditorExportPlatform {
 	Mutex server_lock;
 	Thread server_thread;
 
-	String _get_template_name(bool p_extension, bool p_debug) const {
+	String _get_template_name(bool p_extension, bool p_compatible, bool p_debug) const {
 		String name = "web";
 		if (p_extension) {
 			name += "_dlink";
 		}
+		if (p_compatible) {
+			name += "_compatible";
```

Comment:
Yeah I'm not fond of that name. Why would one NOT want a "compatible" version?

I think "nothreads" could work.